products/kolibri.md

```diff
@@ -0,0 +1,168 @@
+  - purl: pkg:npm/%40public-ui/angular-v21
+  - purl: pkg:npm/%40public-ui/hydrate
+  - purl: pkg:npm/%40public-ui/kolibri-cli
+  - purl: pkg:npm/%40public-ui/mcp
+  - purl: pkg:npm/%40public-ui/react
+  - purl: pkg:npm/%40public-ui/react-v19
+  - purl: pkg:npm/%40public-ui/preact
+  - purl: pkg:npm/%40public-ui/sample-react
+  - purl: pkg:npm/%40public-ui/solid
+  - purl: pkg:npm/%40public-ui/svelte
+  - purl: pkg:npm/%40public-ui/theme-bwst
+  - purl: pkg:npm/%40public-ui/theme-default
+  - purl: pkg:npm/%40public-ui/theme-ecl
+  - purl: pkg:npm/%40public-ui/visual-tests
+  - purl: pkg:npm/%40public-ui/vue
+
+auto:
+  methods:
+    - npm: "@public-ui/components"
+
+releases:
+  - releaseCycle: "5"
+    releaseDate: 2026-12-01
+    eoas: false
+    eol: 2027-12-31
+    latest: "5.0.0"
+    latestReleaseDate: 2026-12-01
+    supportedPhpVersions: STS
+
+  - releaseCycle: "4"
+    releaseDate: 2025-12-01
+    eoas: false
+    eol: 2028-12-31
+    latest: "4.0.0"
+    latestReleaseDate: 2025-12-23
+    lts: true
+
+  - releaseCycle: "3"
+    releaseDate: 2024-12-01
+    eoas: 2025-12-23
+    eol: 2025-12-31
+    latest: "3.0.9"
+    latestReleaseDate: 2025-12-18
+
+  - releaseCycle: "2"
+    releaseDate: 2023-12-01
+    eoas: 2024-12-01
+    eol: 2026-12-31
+    latest: "2.2.18"
+    latestReleaseDate: 2025-12-05
+    lts: true
+
+  - releaseCycle: "1"
+    releaseDate: 2021-12-01
+    eoas: 2023-12-01
+    eol: 2024-12-31
+    latest: "1.7.10"
+    latestReleaseDate: 2024-03-15
+    lts: true
+
+  - releaseCycle: "0"
+    releaseDate: 2020-07-01
+    eoas: 2021-12-01
+    eol: 2021-12-31
+    latest: "0.9.0"
+    latestReleaseDate: 2021-11-15
+
+---
+
+> [KoliBri](https://public-ui.github.io/) (Component Library for Accessibility) is an accessible
+> Web Component Library developed by ITZBund (Federal Information Technology Center of Germany).
+> It provides semantically accessible UI components certified according to WCAG 2.1 and BITV 2.0
+> that can be used with any web framework.
+
+KoliBri follows [Semantic Versioning](https://semver.org/) principles. The library is primarily
+developed for use in government agencies and public institutions but is available as open source
+under the [EUPL-1.2 License](https://github.com/public-ui/kolibri/blob/main/LICENSE).
+
+## Support Policy
+
+KoliBri uses a tiered support model with **STS** (Short-Term Support) and **LTS** (Long-Term Support)
+releases to balance innovation with stability for enterprise deployments.
+
+### Support Types
+
+| Type | Duration | Description |
+|------|----------|-------------|
+| **STS** | 15 months | Short-Term Support releases receive active development with new features, improvements, and all bug fixes. Recommended for projects that want the latest capabilities. |
+| **LTS** | 36 months | Long-Term Support releases receive critical security fixes and important bug fixes only. Recommended for production environments requiring maximum stability. |
```

Comment:
Better to use plain text instead of tables here. 

```suggestion
- **Short-Term Support** releases are supported for 15 months. STS releases receive active develpment with new features. Recommended for projects that want the latest capabilities.
- **Long-Term Support** releases are supported for 3 years. They receive critical security fixes and important bug fixes only. LTS releases are recommended for production environments requiring maximum stability.
```